src/modules/cmdpal/Microsoft.CmdPal.UI/ErrorReportWindow.xaml

```diff
@@ -0,0 +1,101 @@
+<Window
+    x:Class="Microsoft.CmdPal.UI.ErrorReportWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    Title="Command Palette - Error Report"
+    Closed="ErrorReportWindow_OnClosed"
+    >
+
+    <Grid x:Name="RootGrid"
+          RequestedTheme="Default">
+
+        <Grid.Resources>
+            <Thickness x:Key="WindowPadding">24</Thickness>
+            <x:Double x:Key="ElementSpacing">8</x:Double>
+        </Grid.Resources>
+
+        <Grid RowDefinitions="*,Auto">
```

Comment:
Setting RowDefinitions/ColumnDefinitions like this will break hot-reloading, so for a better dev experience, we should be defining these as:

`<Grid.RowDefinitions>
  <RowDefinition...`